classes/question/bank/question_bank_filter.php

```diff
@@ -444,6 +444,27 @@ public function get_sql_filter($data) {
         }
         return array($res, $params);
     }
+
```

Comment:
I know that the general policy is to not comment overridden methods. However, it seems that in StudentQuiz, they consistently document overridden methods, to keep moodle-plugin-ci passing. So, we should probably follow that.